pkg/storage/pod.go

```diff
@@ -62,10 +62,12 @@ func (s *podStorage) GetMetrics(pods ...apitypes.NamespacedName) ([]api.TimeInfo
 			cms              = make([]metrics.ContainerMetrics, 0, len(lastPod.Containers))
 			earliestTimeInfo api.TimeInfo
 		)
+		getPodMetrics := true
```

Comment:
Hi, @dgrisonnet  Thanks for your suggesion, I have updated the code. Could you please take a look again? thanks.

Reply:
Please propose a name that reflects what the variable means, not how we are using it. Name like "allContainersPresent" or "noContainersSkipped" would make this easier to understand.